Vagrantfile.old

```diff
@@ -1,310 +0,0 @@
-
-    # Configure some fancy NFS syncing
-    override.vm.synced_folder ".", "/home/vagrant/.hoot-nfs", type: "nfs",
-      :mount_options => ['vers=3','udp','noatime','nodiratime','nocto', 'nolock'],
-      :linux__nfs_options => ['rw','no_subtree_check','all_squash','async']
-    override.bindfs.bind_folder "/home/vagrant/.hoot-nfs",
-      "/home/vagrant/hoot",
-      chgrp_ignore: true,
-      chown_ignore: true,
-      perms: nil
-    libvirt.cpu_model = "host-passthrough"
-    #override.vm.box = "s3than/trusty64"
-    #override.vm.box_url = "https://app.vagrantup.com/s3than/boxes/trusty64"
-    #override.vm.box = "iknite/trusty64"
-    #override.vm.box_url = "https://app.vagrantup.com/iknite/boxes/trusty64"
-    libvirt.memory = 8192
-    libvirt.cpus = 8
-  end
-
-  # This is a provider for the Parallels Virtualization Software
-  # Run "vagrant up --provider=parallels" to spin up using parallels.
-  # WARNING: Minimally tested
-  config.vm.provider "parallels" do |para, override|
-    para.memory = 8192
-    para.cpus = 4
-    override.hoot_ubuntu1604.vm.box = "parallels/ubuntu-16.04"
-    override.hoot_ubuntu1604.vm.box_url = "https://atlas.hashicorp.com/parallels/boxes/ubuntu-16.04"
-    override.hoot_centos7.vm.box = "parallels/centos-7.3"
-    override.hoot_centos7.vm.box_url = "https://atlas.hashicorp.com/parallels/boxes/centos-7.3"
-  end
-
-  # This is a provider for VMware Workstation
-  # Run "vagrant up --provider=vmware_workstation" to spin up using VMware.
-  # WARNING: Minimally tested
-  config.vm.provider "vmware_workstation" do |vw, override|
-    vw.memory = 8192
-    vw.cpus = 4
-    override.hoot_ubuntu1604.vm.box = "puphpet/ubuntu1604-x64"
-    override.hoot_ubuntu1604.vm.box_url = "https://atlas.hashicorp.com/puphpet/boxes/ubuntu1604-x64"
-  end
-
-  # VSphere provider
-  config.vm.provider "vsphere" do |vsphere, override|
-    override.vm.box     = 'VSphereDummy'
-    override.vm.box_url = 'VSphereDummy.box'
-    vsphere.insecure    = true
-    vsphere.cpu_count   = 8
-    vsphere.memory_mb   = 16384
-    if ENV.key?('VSPHERE_HOST')
-      vsphere.host = ENV['VSPHERE_HOST']
-    end
-    if ENV.key?('VSPHERE_RESOURCE')
-      vsphere.compute_resource_name = ENV['VSPHERE_RESOURCE']
-    end
-    if ENV.key?('VSPHERE_PATH')
-      vsphere.vm_base_path = ENV['VSPHERE_PATH']
-    end
-    if ENV.key?('VSPHERE_TEMPLATE')
-      vsphere.template_name = ENV['VSPHERE_TEMPLATE']
-    end
-    if ENV.key?('VSPHERE_USER')
-      vsphere.user = ENV['VSPHERE_USER']
-    end
-    if ENV.key?('VSPHERE_PASSWORD')
-      vsphere.password = ENV['VSPHERE_PASSWORD']
-    end
-    if ENV.key?('VSPHERE_NAME')
-      vsphere.name = ENV['VSPHERE_NAME']
-    end
-  end
-end
-
-# Allow local overrides of vagrant settings
-if File.exists?('VagrantfileLocal')
-  load 'VagrantfileLocal'
-else
-  if File.exists?('VagrantfileLocal.vbox')
-    load 'VagrantfileLocal.vbox'
-  end
-end
```

Comment:
I deleted this file as it is an old file and a `.old` file that we don't seem to need anymore.